tests/test_bukuDb.py

```diff
@@ -1055,6 +1055,54 @@ def custom_fetch(url, http_head=False):
               '        <DD>The official home...',
               '    </DL><p>',
               '</DL><p>']),
+    ('rss', ['<feed xmlns="http://www.w3.org/2005/Atom">',
+             '    <title>Bookmarks</title>',
+             '    <generator uri="https://github.com/jarun/buku">buku</generator>',
+             '    <entry>',
+             '        <title>Fetched Title (DELETED)</title>',
+             '        <link href="http://custom.url" rel="alternate" type="text/html"/>',
+             '        <id>1</id>',  # deleted entry, pre-deletion ID
+             '        <content type="html"> <![CDATA[ <p>Fetched description.</p> ]]> </content>',
+             '    </entry>',
+             '    <entry>',
+             '        <title>Wikipedia (OLD URL = http://wikipedia.net)</title>',
+             '        <link href="https://www.wikipedia.org" rel="alternate" type="text/html"/>',
+             '        <id>1</id>',  # updated entry, current ID
```

Comment:
Not sure if this is actually considered valid data but that's what we get in the input… Changing it would probably require coming up with some custom ID generation logic for RSS export (for cases when duplicate IDs are detected in the input, I guess?…).

`.db` format produces IDs `1`, `2` & `3` respectively, but _a)_ duplicate IDs would be rejected by the library, and _b)_ I'm pretty sure it ignores input IDs to begin with :sweat_smile: